### PR TITLE
pythonPackages.filterpy: init at 1.4.5

### PR DESCRIPTION
**Pull request by costrouc**
###### Things done

pythonPackages.filterpy: init at 1.4.5 

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [X] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by FRidh** [comment]
@GrahamcOfBorg build python3Packages.filterpy

File: pkgs/development/python-modules/filterpy/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  src = fetchPypi {
+    inherit pname version;
+    extension = "zip";
+    sha256 = "4f2a4d39e4ea601b9ab42b2db08b5918a9538c168cff1c6895ae26646f3d73b1";
+  };
+
+  checkInputs = [ pytest ];
+  propagatedBuildInputs = [ numpy scipy matplotlib ];
+
+  # single test fails
```
Comment (Infinisil) [inline pkgs/development/python-modules/filterpy/default.nix]: Why does it fail?

Comment (costrouc) [reply pkgs/development/python-modules/filterpy/default.nix]: I tested the master branch without nix and that test fails then too. Also the repository has tests but does not run then with CI. I have added a better comment to explain why the test fails.

**Comment by Infinisil** [comment]
Oh actually, this seems to even be the only test: https://github.com/rlabbe/filterpy/tree/master/filterpy/common/tests

So you might as well just do `doCheck = false`

**Comment by costrouc** [comment]
@Infinisil there are actually about 50 tests if I remember correctly. The tests are within each folder for example https://github.com/rlabbe/filterpy/tree/master/filterpy/kalman/tests.

**Comment by Infinisil** [comment]
Ah I see

@GrahamcOfBorg build python2Packages.filterpy python3Packages.filterpy

**Pull request merged**